inst/templates/qData-prep.R

```diff
@@ -3,6 +3,7 @@
 # This is a template for importing, cleaning, and exporting data
 # ready for the qPackage.
 library(qData)
+library(qCreate)
```

Comment:
Do we need to load qData if we're loading qCreate (which should load qData anyway, no?)